src/serac/physics/utilities/quadrature_data.hpp

```diff
@@ -200,12 +259,15 @@ class QuadratureData : public SyncableData {
  */
 template <>
 class QuadratureData<void> {
+  // Doesn't do anything
+public:
+  SERAC_HOST_DEVICE std::nullptr_t operator()(const int, const int) { return nullptr; }
 };
 
 // A dummy global so that lvalue references can be bound to something of type QData<void>
 // FIXME: There's probably a cleaner way to do this, it's technically a non-const global
 // but it's not really mutable because no operations are defined for it
-extern QuadratureData<void> dummy_qdata;
+extern SERAC_DEVICE QuadratureData<void> dummy_qdata;
```

Comment:
does this need to be annotated with `SERAC_DEVICE`? see:

```
sam@provolone:~/code/cuda_sandbox$ cat global_in_kernel.cu 
#include <cstdio>

template < typename T >
struct QuadratureData{
  __host__ __device__ T operator()(int i, int j) { 
    printf("calling QuadratureData<T>::operator() on device\n");
    return T{}; 
  }
};

QuadratureData<double> dummy;

__global__ void kernel(QuadratureData<double> data) {
  double value = data(0, 0);
}

int main() {
  kernel<<<1,1>>>(dummy);
  cudaDeviceSynchronize();
}
sam@provolone:~/code/cuda_sandbox$ nvcc global_in_kernel.cu 
sam@provolone:~/code/cuda_sandbox$ cuda-memcheck a.out 
========= CUDA-MEMCHECK
calling QuadratureData<T>::operator() on device
========= ERROR SUMMARY: 0 errors
```

Reply:
I think this must have been an artifact from earlier in the development process, thanks!